repos/system_upgrade/common/actors/cloud/checkrhui/actor.py

```diff
@@ -68,8 +72,13 @@ def process(self):
                     self.produce(DNFPluginTask(name='amazon-id', disable_in=['upgrade']))
                 # if RHEL7 and RHEL8 packages differ, we cannot rely on simply updating them
                 if info['el7_pkg'] != info['el8_pkg']:
-                    self.produce(RpmTransactionTasks(to_install=[info['el8_pkg']]))
-                    self.produce(RpmTransactionTasks(to_remove=[info['el7_pkg']]))
-                self.produce(RHUIInfo(provider=provider))
-                self.produce(RequiredTargetUserspacePackages(packages=[info['el8_pkg']]))
+                    self.produce(RpmTransactionTasks(
+                        to_install=[info['el8_pkg']],
```

Comment:
Since this is in `common`, do we want to keep `el8_pkg` and `el7_pkg`? Renaming those to `old_pkg` and `new_pkg` would make much more sense in `common` context.

Reply:
@shaded-enmity Good point. We could define the new/old variables in the begining of the begining and than just use these variables everywhere to work with the map in the share library.